docs/dictionary.rst

```diff
@@ -14,8 +14,9 @@ Universal domain names
 * Nazev stranky / rubriky = Page Heading
 * Nickname = nick (always shortened)
 * Anotace = synopsis
-* Prispevek = Creation
+* Prispevek / dílo = Creation
 * Koncepce (rubriky) = Charter
+* Statistiky = User profile
```

Comment:
User Profile has other meaning (in the code). 

Would User Stats work?

Reply:
@Almad sure

Reply:
@Nathaka you marked it as resolved, but I don't see it reflected :)